examples/specs/box_plot_minmax.vl.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I think you would want to rename this fix to

examples/specs/box_plot_minmax_full.vl.json

and create a new 

examples/specs/box_plot_minmax_short.vl.json

using your new syntax :)